source/getstarted.html.erb

```diff
@@ -0,0 +1,84 @@
+  <nav class="breadcrumbs" aria-label="Breadcrumbs">
+      <ol itemscope itemtype="http://schema.org/BreadcrumbList">
+          <li class="breadcrumbs__item" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+              <a href="https://www.gov.uk/service-toolkit#components" itemprop="item"><span itemprop="name">Components</span></a>
+          </li>
+          <li class="breadcrumbs__item breadcrumbs__item" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+              <a href="/" itemprop="item"><span itemprop="name">GOV.UK Pay</span></a>
+          </li>
+          <li class="breadcrumbs__item breadcrumbs__item--active" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+              <a href="#main" itemprop="item"><span itemprop="name">Get Started</span></a>
+          </li>
+      </ol>
+  </nav>
+<div class="container">
+  <div class="grid-row">
+      <div class="column-two-thirds column-minimum">
+        <h1 class="heading-large">
+          Get started
+        </h1>
+
+        <ol class="task-list">
+            <h2 class="task-list-section">
+              <span class="task-list-section-number">1.</span> Check GOV.UK Pay is right for you
+            </h2>
+            <div class="task-list-items">
+              <p>If you are a central government organisation with a new or existing digital service, and want to take card payments online, then GOV.UK&nbsp;Pay should be suitable for you.</p>
+              <p>Review GOV.UK Pay&rsquo;s <a href="/features.html">top features and roadmap</a> and decide if it meets your service&rsquo;s needs.</p>
```

Comment:
Again, should be able to just use the _’_ character directly (applies to a few places in this file)